lib/cli/commands/listorders.ts

```diff
@@ -89,20 +78,32 @@ const displayTables = (orders: ListOrdersResponse.AsObject) => {
   formatOrders(orders).forEach(displayOrdersTable);
 };
 
-export const command = 'listorders [pair_id]';
+export const command = 'listorders [pair_id] [include_own_orders] [limit]';
 
 export const describe = 'list orders from the order book';
 
 export const builder = {
   pair_id: {
+    describe: 'pair id for which to retrieve orders.',
     type: 'string',
   },
+  include_own_orders: {
+    describe: 'should include own orders',
```

Comment:
"whether to include own orders" assuming that still fits on one line in the help output.